cmake/FindLibUV.cmake

```
@@ -75,6 +75,13 @@ if(WIN32)
   list(APPEND LIBUV_LIBRARIES ws2_32)
 endif()
 
+if(Threads_FOUND)
+  # TODO: Fix the cmake file to properly handle static deps for bundled builds,
+  # but for not just include the threads library if CMake tells us there's one
```

"now" has a confusing typo, so i will also editorialize :)
```suggestion
  # TODO: Fix the cmake file to properly handle static deps for bundled builds.
  # Meanwhile just include the threads library if CMake tells us there's one
```